apps/heft/package.json

```diff
@@ -62,6 +62,6 @@
     "@types/semver": "7.3.5",
     "colors": "~1.2.1",
     "tslint": "~5.20.1",
-    "typescript": "~4.8.4"
```

Comment:
How stable is 5.0.4? It seems kind of aggressive to be upgrading right away.

Reply:
Well, 5.1 beta is already out =P
https://devblogs.microsoft.com/typescript/announcing-typescript-5-1-beta/